src/api/telemetry/TelemetryAPI.js

```diff
@@ -489,6 +489,62 @@ export default class TelemetryAPI {
     }.bind(this);
   }
 
+  /**
+   * Subscribe to limits for a specific domain object.
```

Comment:
Maybe we should specifically say `Subscribe to run-time changes in configured telemetry limits` here. Developers might be confused and think we are subscribing to limit violations, which we're not.

Reply:
Done